src/main/kotlin/com/hibob/kotlinBasic/basicOfkotlin.kt

```diff
@@ -0,0 +1,38 @@
+package com.hibob.kotlinBasic
+fun main() {
+    helloWorld()
+    evenOrOdd(3)
+    println(checkEqual(4,3))
+    println(checkEqual(4,4))
+    max(5,4)
+
+    // a=1 b=5
+    println( multiplication(b=5))
```

Comment:
space is reduntant, Its not critical for the code but we want to keep on code style.
its relevant to all of the below println.

I suggest you to use cmd + option + L, its automaticlly fix the style